pkg/kanctl/actionset.go

```diff
@@ -342,6 +356,28 @@ func parseProfile(cmd *cobra.Command, ns string) (*crv1alpha1.ObjectReference, e
 	}, nil
 }
 
+// parseRepositoryServer returns the object reference
+// for the passed Repository Server
+func parseRepositoryServer(cmd *cobra.Command) (*crv1alpha1.ObjectReference, error) {
+	repositoryServerName, _ := cmd.Flags().GetString(repositoryServerFlagName)
+	if repositoryServerName == "" {
+		return nil, nil
```

Comment:
should we return an error here or an error message describing the issue instead of returning nil?

Reply:
We're handling errors where we're calling this function. I think we're doing similar while fetching other CRs. Let's see what others say.

Reply:
It's not about handling errors, even if we are handling errors where we are calling this function from; that would be of no use because we are not returning error.
Again, handling errors comes into picture when we return error.
One thing that I can think of is, we don't necessarily want to fail the `kanctl` command when repo server is not specified because there are chances that profile is specified. 
Can you please point me to the code where this check is performed to make sure either profile or repo server is  specified.

Reply:
@viveksinghggits, While creating actionsets, we don't check/verify that either `--profile` or `--repository-server` flag is passed. in case either is passed, only then, it would verify the resources.

Reply:
Added check, if `--profile` or `--repository-server` flag is not provided, `kanctl` will warn user but will create actionset